internal/handlers/images.go

```diff
@@ -130,7 +121,24 @@ func (h *ImageHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	isoReader, err := h.imageStreamForID(r.Context(), clusterID, params)
```

Comment:
Just wonder if we should keep the imageStreamForID func, as ServeHTTP gets a little big. Or you prefer to avoid logging from another func, etc.

Reply:
:man_shrugging: it didn't seem like it was adding much.